### PR TITLE
Get old/default column value for building rowset_update_state in the write phrase

### PR DESCRIPTION
**Pull request by sevev**
This pr will get the old/default column values to create the rowset_update_state currently created in the apply phrase in the  write phrase of partial update
Build rowset_update_state in write phrase can reduce the time of apply

For https://github.com/StarRocks/starrocks/issues/2891

### CODE REVIEW

**Comment by sevev** [comment]
run starrocks_be_unittest

File: be/src/storage/rowset/beta_rowset_writer.cpp [hunk 1]
```diff
@@ -144,6 +145,29 @@ StatusOr<RowsetSharedPtr> BetaRowsetWriter::build() {
     RETURN_IF_ERROR(
             RowsetFactory::create_rowset(_context.tablet_schema, _context.rowset_path_prefix, _rowset_meta, &rowset));
     _already_built = true;
+
+    // if rowset is partial rowset, build rowset_update_state in advance to reduce cost time of apply
```
Comment (chaoyli) [inline be/src/storage/rowset/beta_rowset_writer.cpp:149]: This pr seems to construct parial_update_state not get the old default column.
The content is not consistent with the title.

Comment (sevev) [reply be/src/storage/rowset/beta_rowset_writer.cpp:149]: > This pr seems to construct parial_update_state not get the old default column. The content is not consistent with the title.

The purpose of getting old/default value is to construct partial_update_state, changed the title to make it easier to understand

**Comment by sevev** [comment]
rowset_update_state has been built in funciton `UpdateManager::on_rowset_finised` before rowset commit, this pr repeats the implementation of this function and can be closed

**Pull request closed without merging**